File: submissions/sm_022_prakash/week_03/day_3/submissions_2/average_divisible_while.js
sum = 0;
count = 0;
//function to print average of all the numbers divisible by given number
function average(number,limit) {
  for (i = 1; i <= limit; i++) { 
    while (i % number == 0) {
      sum = sum + i;
      count = count + 1  
      break;
    }
    average = sum / count;
  }
  return average;
}
//calling the function 
average(3,20);
console.log(average);

File: submissions/sm_102_amit/week_13/day_4/session_1/average_divisible_while.py
# Print the average of all the numbers from 0 to the given limit that are divisible by the given number
limit = input("Enter limit: ")
num = input("Enter number: ")
sum = 0
for i in range(0, int(limit) + 1, int(num)):
    sum += i
avg = sum / int(limit)
print(avg)
